bindings/nodejs/client_context.cpp

```diff
@@ -0,0 +1,64 @@
+#include "client_context.hpp"
+
+#include "hawktracer/client_utils/stream_factory.hpp"
+#include "hawktracer/parser/make_unique.hpp"
+#include <iostream>
+#include <utility>
+
+namespace HawkTracer
+{
+namespace Nodejs
+{
+
+std::unique_ptr<ClientContext> ClientContext::create(const std::string &source, EventCallback event_callback)
+{
+    std::unique_ptr<parser::Stream> stream = ClientUtils::make_stream_from_string(source);
+    if (!stream) {
+        return nullptr;
+    }
+
+    std::unique_ptr<parser::KlassRegister> klass_register{new parser::KlassRegister()};
```

Comment:
Done in [4f074](https://github.com/amzn/hawktracer/pull/67/commits/4f07400dc79d7288fe687268ec6b6c2b87908d85), but I couldn't use it for `ClientContext` since its constructor is not exposed.